.github/workflows/reusable-zizmor.yml

```
@@ -124,16 +129,24 @@ jobs:
           --version "${ZIZMOR_VERSION}"
           zizmor
 
+      - name: Zizmor cache
+        uses: actions/cache@5a3ec84eff668545956fd18022155c47e93e2684 # v4
+        with:
+          path: ${{ runner.temp }}/.cache/zizmor
+          key: zizmor-${{ runner.os }}-${{ runner.arch }}-${{ hashFiles('**/.github/workflows/*', '**/action.{yaml,yml}') }}
```

Hmm, I think this means we'll throw the cache away whenever the actions change.

```suggestion
          key: zizmor-${{ runner.os }}-${{ runner.arch }}
```

I wonder what the impact of having a more general cache key would be? Feels to me like previously cached values would still be valid - it'll be requests/responses made to the GitHub API and those should be good to be cached even if the source of them is different.

---

That makes a lot of sense